tests/gdx-tests/src/com/badlogic/gdx/tests/extensions/ControllersTest.java

```diff
@@ -1,184 +1,283 @@
-/*******************************************************************************
```

Comment:
@GreenLightning 
That's the problem. The `core.autocrlf` doesn't work for me (and you)